File: JavaScript/Iniciante/1011.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

raio = parseFloat(lines.shift());
pi = 3.14159;

volume = (4.0/3.0) * pi * Math.pow(raio, 3);

console.log(`VOLUME = ${volume.toFixed(3)}`);

File: GO/Iniciante/1011.go
package main

import (
	"fmt"
  "math"
)

func main() {

  var raio float64
  pi := 3.14159
  fmt.Scan(&raio)
  
  volume := (4.0/3.0) * pi * math.Pow(raio, 3)

  fmt.Printf("VOLUME = %.3f\n", volume)
  
}
